arch/cpu/arm/cortex-m/Makefile.cortex-m

```diff
@@ -1,4 +1,4 @@
-CONTIKI_ARM_DIRS += cortex-m cortex-m/CMSIS
+CONTIKI_ARM_DIRS += cortex-m CMSIS/CMSIS/Core/Include
```

Comment:
and if we do the above then this becomes `CMSIS/Core/Include`, which is a bit prettier